build/build_android_llm_demo.sh

```diff
@@ -97,7 +97,7 @@ build_aar() {
   find jni -type f -name "libexecutorch_jni.so" -exec bash -c 'mv "$1" "${1/_jni/}"' bash {} \;
   # Zip all necessary files into the AAR file
   zip -r executorch.aar libs jni/*/libexecutorch.so jni/*/libqnn*.so jni/*/libQnn*.so AndroidManifest.xml
-  zip -r executorch-llama.aar libs jni/*/libexecutorch.so jni/*/libqnn*.so jni/*/libQnn*.so AndroidManifest.xml
+  cp executorch.aar executorch-llama.aar
```

Comment:
It's separate change... But I will get rid of executorch-llama.aar later. It's the same as executorch.aar as of now. I don't have time to clean up stuff